src/features/MovieDetails/Header/styled.js

```diff
@@ -0,0 +1,118 @@
+import styled from "styled-components";
+import poster from "./poster.png";
+
+export const WrapperPoster = styled.div`
+margin: 0 auto;
+    max-width: 1920px;
+    max-height: 769px;
+    height: 50vw;
+    background-image: url("${poster}");
+    background-repeat: no-repeat;
+    background-position: center;;
+    background-size: cover;
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.medium}px) {
+        max-height:600px;
+};
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.small}px) {
+        max-height: 400px;
+    };
+`;
+
+export const Wrapper = styled.div`
+    max-width: 1920px;
+    width: 100vw;
+    max-height: 769px;
+    height: 50vw;
+    background-repeat: no-repeat;
+    display:flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    background: linear-gradient(270deg, #000000 14.11%,
+                    rgba(0, 0, 0, 0.873268) 15.08%,
+                    rgba(0, 0, 0, 0.720529) 16.51%,
+                    rgba(0, 0, 0, 0.294577) 19.99%,
+                    rgba(0, 0, 0, 0.159921) 21.88%,
+                    rgba(0, 0, 0, 0) 25.68%),
+                linear-gradient(90deg, #000000 13.6%,
+                    rgba(0, 0, 0, 0.984059) 14.58%,
+                    rgba(0, 0, 0, 0.967732) 15.44%,
+                    rgba(0, 0, 0, 0.865569) 16.3%,
+                    rgba(0, 0, 0, 0.230315) 22.87%,
+                    rgba(0, 0, 0, 0) 26.64%),
+                linear-gradient(180deg, #000000 0%,
+                    rgba(0, 0, 0, 0.689555) 4.66%,
+                    rgba(0, 0, 0, 0.439033) 9.34%,
+                    rgba(0, 0, 0, 0.20628) 15.16%,
+                    rgba(0, 0, 0, 0) 24.22%),
+                linear-gradient(189.44deg,
+                    rgba(0, 0, 0, 0) 58.48%,
+                    rgba(0, 0, 0, 0.106473) 63.17%,
+                    rgba(0, 0, 0, 0.235359) 68.85%,
+                    rgba(0, 0, 0, 0.492821) 78.08%,
+                    rgba(0, 0, 0, 0.740286) 85.86%, #000000 92.87%);
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.medium}px) {
+        width: 100%;
```

Comment:
I suggest delete this entire passage. It's unnecessary. The gradient used in @media looks better and there is no jump when reducing the screen. On the other hand, no one will be doing it except for testing.